core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/BenchmarkArgs.scala

```diff
@@ -0,0 +1,29 @@
+package org.apache.spark.rapids.tool.benchmarks
+
+import org.rogach.scallop.{ScallopConf, ScallopOption}
+
+class BenchmarkArgs(arguments: Seq[String]) extends ScallopConf(arguments) {
+
+    banner("""
+Benchmarker class for running various benchmarks.
+    """)
+
+  val iterations: ScallopOption[Int] = opt[Int](name = "iterations", short = 'i', default = Some(5),
+    descr = "Total iterations to run")
+
+  val warmupIterations: ScallopOption[Int] = opt[Int](name = "warmupIterations", short = 'w' ,
+    default = Some(3), descr = "Number of warmup iterations to run")
+
+  val outputDirectory: ScallopOption[String] = opt[String](name = "outputDirectory", short = 'd',
+    default = Some("."), descr = "Directory to write output to")
```

Comment:
same here, more details

```
"Base output directory. Default is current directory for the default filesystem. +
        " The final output will go into a subdirectory called" +
        " rapids-tools-benchmark. It will overwrite any existing directory" +
        " with the same name.",
```

Reply:
Added more details